### PR TITLE
LTD-2592-add-name-non-firearms 

### PR DESCRIPTION
**Pull request by depsiatwal**
lot more work to be done here for example 
make all reusable components generic and refactor

This is first draft kind of framework

### CODE REVIEW

File: core/summaries/formatters.py [hunk 1]
```diff
@@ -51,6 +51,17 @@
     "section-5-certificate-missing-reason": "Explain why you do not have a section 5 letter of authority",
 }
 
+COMPLETE_PRODUCT_LABELS = {
```
Comment (kevincarrogan) [inline core/summaries/formatters.py:54]: What does "COMPLETE_PRODUCT" actually mean, it seems like a confusing name for this?

File: exporter/applications/summaries.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
     document_formatter,
 )
 from core.summaries.summaries import firearm_product_summary as core_firearm_product_summary
+from core.summaries.summaries import complete_product_summary as core_complete_product_summary
```
Comment (kevincarrogan) [inline exporter/applications/summaries.py:7]: You don't need the two separate `from` lines, you can do:

```
from core.summaries.summaries import (
    firearm_product_summary as core_firearm_product_summary,
    complete_product_summary as core_complete_product_summary,
)
```

File: exporter/applications/views/goods/add_good_complete_product/views/add.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            },
+        )
+        ctx["title"] = form.Layout.TITLE
+
+        return ctx
+
+    def get_payload(self, form_dict):
+        good_payload = AddGoodCompleteProductPayloadBuilder().build(form_dict)
+        payload = good_payload
+        return payload
```
Comment (kevincarrogan) [inline exporter/applications/views/goods/add_good_complete_product/views/add.py:72]: You don't really need to reassign to a variable in between here.

File: exporter/goods/services.py [hunk 4]
```diff
@@ -299,3 +299,13 @@ def get_good_document_sensitivity(request, pk):
...
+
+
+def post_complete_product(request, json):
+    json["item_category"] = PRODUCT_CATEGORY_PLATFORM
+    json["is_good_controlled"] = None
+    data = client.post(request, "/goods/", json)
+
+    if data.status_code == HTTPStatus.OK:
+        data.json().get("good"), data.status_code
+    return data.json(), data.status_code
```
Comment (kevincarrogan) [inline exporter/goods/services.py:311]: In the view you're doing:

```
            good, _ = self.post_complete_product(form_dict)
            self.good = good["good"]
```
so why are you doing this `.get("good")`.

Is `HTTPStatus.OK` what you're expecting to get back from a `post`?

File: core/summaries/formatters.py (hunk 1 above)
Comment (depsiatwal) [reply core/summaries/formatters.py]: renamed to PLATFORM which is what i'm calling this 'product' for now

File: exporter/applications/summaries.py (hunk 2 above)
Comment (depsiatwal) [reply exporter/applications/summaries.py]: done

File: exporter/applications/views/goods/add_good_complete_product/views/add.py (hunk 3 above)
Comment (depsiatwal) [reply exporter/applications/views/goods/add_good_complete_product/views/add.py]: simplified for now

File: exporter/goods/services.py (hunk 4 above)
Comment (depsiatwal) [reply exporter/goods/services.py]: legacy copy fixed up

**Review by kevincarrogan — APPROVED** [review]

**Pull request merged**